docs/howto/cylc-ui.md

```diff
@@ -1,60 +1,113 @@
 # Getting Started with Cylc 8
 
+## Development Installation
+
+> **Note**: We recommend developing in a Conda environment as this provides a
+  more standard setup though if you ensure the non-python dependencies
+  are installed correctly you can bypass this.
+
 1. Create a clean conda install:
   ```bash
-  conda create -n my-example-env python=3.7 --yes
+  conda create -n my-example-env python=3.7 configurable-http-proxy --yes
   conda activate my-example-env
   ```
 
-2. Ensure that you install `nodejs` using conda first*. If you allow pip to
-  install it as a dependency of cylc & cylc-uiserver it you will end up with
-  a very old version.
-  ```bash
-  conda install nodejs --yes
-  node --version
-  ```
-  For comparison devs have used node at 10.13.x and 12.x.x.
+2. Ensure that you install `nodejs` using conda first.
+
+   If you allow pip to
+   install it as a dependency of cylc & cylc-uiserver it you will end up with
+   a very old version.
+   ```bash
+   conda install nodejs --yes
+   node --version
+   ```
+   For comparison devs have used node at 10.13.x and 12.x.x.
 
 3. Install
    [cylc/cylc-flow](https://github.com/cylc/cylc-flow/) and
    [cylc/cylc-uiserver](https://github.com/cylc/cylc-uiserver/):
-  ```bash
-  pip install -e "${DEVDIR}/cylc-flow"[all]
-  pip install -e "${DEVDIR}/cylc-uiserver"[all]
-  ```
-  where `${DEVDIR}/cylc-flow` and `${DEVDIR}/cylc-uiserver` are the locations
-  of your local (git) worktrees of the two projects.
+
+   ```bash
+   pip install -e "${DEVDIR}/cylc-flow"[all]
+   pip install -e "${DEVDIR}/cylc-uiserver"[all]
+   ```
+
+   Where `${DEVDIR}/cylc-flow` and `${DEVDIR}/cylc-uiserver` are the locations
+   of your local (git) worktrees of the two projects.
 
 4. Install [cylc/cylc-ui](https://github.com/cylc/cylc-ui/).
-  ```bash
-  cd $DEVDIR/cylc-ui
-  npm install --prefix="${DEVDIR}/cylc-ui"
-  npm install -g configurable-http-proxy
-  ```
-  where `${DEVDIR}/cylc-ui` is the location
-  of your local (git) worktree of the project.
-  
+
+   > **Note:** We prefer `yarn`.
+
+   ```bash
+   cd $DEVDIR/cylc-ui
+   yarn install
```

Comment:
Commenting on this as I have been through this recently rebuilding environment from scratch on MO computer. We don't get yarn as standard so, for completeness of instruction... perhaps a warning that might need to do a npm install yarn first?

Reply:
Done.